.hoppscotch/MVP.json

```diff
@@ -0,0 +1 @@
],"method":"PUT","auth":{"authType":"none","authActive":true},"preRequestScript":"","testScript":"// Check status code is 200\npw.test(\"Status code is 200\", ()=> {\n    pw.expect(pw.response.status).toBe(200);\n});\n\n// Check JSON response property\npw.test(\"Check JSON response property\", ()=> {\n    pw.expect(pw.response.body.start).toBeType(\"string\");\n    pw.expect(pw.response.body.stop).toBeType(\"string\");\n});","body":{"contentType":"application/json","body":"{\n    \"start\": \"2023-01-11T17:40:44\",\n    \"stop\": \"2023-01-11T17:40:48\"\n}"}},{"preRequestScript":"","endpoint":"<<host>>/api/items/<<item_id>>/timers/<<timer_id>>","params":[],"v":"1","headers":[{"active":true,"value":"application/json","key":"accept"}],"testScript":"// Check status code is 400\npw.test(\"Status code is 400\", ()=> {\n    pw.expect(pw.response.status).toBe(400);\n});","body":{"body":"{\n    \"start\": \"2023-invalid-01\"\n}","contentType":"application/json"},"name":"Update timer (400 - Invalid date format)","method":"PUT","auth":{"authType":"none","authActive":true}},{"preRequestScript":"","endpoint":"<<host>>/api/items/<<item_id>>/timers/<<timer_id>>","body":{"contentType":"application/json","body":"{\n    \"start\": \"2023-01-11T17:40:44\",\n    \"stop\": \"2023-01-11T17:40:40\"\n}"},"auth":{"authActive":true,"authType":"none"},"headers":[{"key":"accept","active":true,"value":"application/json"}],"v":"1","testScript":"// Check status code is 400\npw.test(\"Status code is 400\", ()=> {\n    pw.expect(pw.response.status).toBe(400);\n});","name":"Update timer (400 - Stop is after start)","params":[],"method":"PUT"},{"name":"Stop timer","method":"PUT","auth":{"authType":"none","authActive":true},"params":[],"v":"1","testScript":"// Check status code is 200\npw.test(\"Status code is 200\", ()=> {\n    pw.expect(pw.response.status).toBe(200);\n});","headers":[{"key":"accept","value":"application/json","active":true}],"body":{"body":null,"contentType":null},"preRequestScript":"","endpoint":"<<host>>/api/timers/<<timer_id_to_stop>>"},{"preRequestScript":"","headers":[{"active":true,"key":"accept","value":"application/json"}],"endpoint":"<<host>>/api/timers/<<timer_id>>","body":{"body":null,"contentType":null},"method":"PUT","name":"Stop timer (403 - Timer has already been stopped)","v":"1","params":[],"testScript":"\n\n// Check status code is 403\npw.test(\"Status code is 403\", ()=> {\n    pw.expect(pw.response.status).toBe(403);\n});","auth":{"authType":"none","authActive":true}},{"preRequestScript":"","body":{"contentType":null,"body":null},"v":"1","method":"PUT","name":"Stop timer (404 - Timer not found)","endpoint":"<<host>>/api/timers/<<notfound_timer_id>>","auth":{"authActive":true,"authType":"none"},"testScript":"// Check status code is 404\npw.test(\"Status code is 404\", ()=> {\n    pw.expect(pw.response.status).toBe(404);\n});","headers":[{"value":"application/json","key":"accept","active":true}],"params":[]}]}],"v":1,"name":"MVP","requests":[]}
```

Comment:
@LuchoTurtle is there a way to keep this `.json` doc "pretty" 🤷‍♂️ 
i.e: not just a single line "blob" that's impossible to review/maintain. 💭

Reply:
I could format it (or add a step in the CI that format this file) but this is the default export from the collections, coming straight from Hoppscotch :/